docs/elements.md

```diff
@@ -133,6 +136,7 @@ incorrect answers and displays them in a random order as radio buttons.
 | `external-json`               | string  | special | Optional path to a JSON file to load external answer choices from. Answer choices are stored as lists under "correct" and "incorrect" key names.            |
 | `external-json-correct-key`   | string  | special | Optionally override default json "correct" attribute name when using `external-json` file.                                                                  |
 | `external-json-incorrect-key` | string  | special | Optionally override default json "incorrect" attribute name when using `external-json` file.                                                                |
+| `allow-blank`                 | boolean | false   | Whether or not an empty submission is allowed. If `allow-blank` is set to `true`, no response will be marked as incorrect instead of invalid.               |
```

Comment:
What is the use case for this? I'm mostly just curious, I don't see why I would ever allow a student to click submit without answering the question.

Reply:
The issue https://github.com/PrairieLearn/PrairieLearn/issues/6350 is for adding `allow-blank` in every element. I actually had some issues trying to include this functionality and wound up getting rid of it, but forgot to remove it from the docs.

Reply:
This is mostly for questions with several elements, including a `pl-multiple-choice`, where a student chooses to deliberately keep it blank (e.g., out of time) but still wants partial grades for previous elements. If one element gives an invalid parse result, the question is fully invalid.